doc/source/whatsnew/v0.22.0.txt

```diff
@@ -188,7 +188,8 @@ Other API Changes
 - :func:`pandas.DataFrame.merge` no longer casts a ``float`` column to ``object`` when merging on ``int`` and ``float`` columns (:issue:`16572`)
 - The default NA value for :class:`UInt64Index` has changed from 0 to ``NaN``, which impacts methods that mask with NA, such as ``UInt64Index.where()`` (:issue:`18398`)
 - Refactored ``setup.py`` to use ``find_packages`` instead of explicitly listing out all subpackages (:issue:`18535`)
-- Rearranged the order of keyword arguments in :func:`read_excel()` to align with :func:`read_csv()` (:pr:`16672`)
+- Rearranged the order of keyword arguments in :func:`read_excel()` to align with :func:`read_csv()` (:issue:`16672`)
+- :func:`DatetimeIndex.to_series` now allows kwargs which are passed to :class:`Series` constructor (:issue:`18699`)
```

Comment:
just same we now only accept kwargs ``name,index`` ``:func;`Index..to_series` ``

Reply:
actually, make this in the bug-fix section and not here